documentation/upgrade.md

```diff
@@ -1,3 +1,8 @@
+# Changes in version 4.0.0
+
+* Operator-sdk updated to v1.3.0
+* Installs Grafana 7.1.1 by default
```

Comment:
We should mention: https://github.com/grafana-operator/grafana-operator/pull/488
Since this is a rather big change in how the grafana-operator works today.